### ISSUE TITLE
Ensure links explain their purpose

### ISSUE THREAD

**Issue by unclesam2024** [web-infrastructure/doc_architecture#54]
The purpose of a link should be clear from the text inside the link. Links like "learn more" are not helpful to users with accessible technology.

Links should also avoid redundantly saying that they are links, e.g. "link to page" as screen readers will read these out with the word "link" added, e.g. "link link to page".

Exceptions to these rules are allowed where the purpose of a link would be ambiguous to users in general.
